src/i18n/messages.ar.xlf

```diff
@@ -154,13 +154,29 @@
         </context-group>
         <target xml:lang="ar" state="needs-translation">Delete</target>
       </trans-unit>
+      <trans-unit id="cb33b150a0b9e6b88a9aa5fa79f91888dd06784a" datatype="html">
+        <source>ltr</source>
+        <context-group purpose="location">
+          <context context-type="sourcefile">app/home/home.component.ts</context>
+          <context context-type="linenumber">2</context>
+        </context-group>
+        <context-group purpose="location">
+          <context context-type="sourcefile">app/home/home.component.ts</context>
+          <context context-type="linenumber">3</context>
+        </context-group>
+        <context-group purpose="location">
+          <context context-type="sourcefile">app/login/login.component.ts</context>
+          <context context-type="linenumber">1</context>
+        </context-group>
+        <target xml:lang="ar" state="needs-translation">rtl</target>
+      </trans-unit>
       <trans-unit id="3f384a474f6473824411ffbd437db55c8a45dad6" datatype="html">
```

Comment:
We've got a problem.  This `id` doesn't match the `id` in `messages.xlf`.  Is there a way that we can quickly confirm that there aren't other ids that have been switched?